### PR TITLE
tse: add websocket timeout config, improve error handling

### PR DESCRIPTION
**Pull request by mic-e**

### CODE REVIEW

File: stustapay/tse/diebold_nixdorf_usb/handler.py [hunk 1]
```diff
@@ -15,33 +16,58 @@
...
+            def set_ws(ws):
+                print(f'setting self._ws={ws}')
+                self._ws = ws
+            try:
+                session = await stack.enter_async_context(aiohttp.ClientSession(timeout=aiohttp.ClientTimeout(total=self.websocket_timeout, connect=self.websocket_timeout)))
+                LOGGER.info(f'connecting to DN TSE at {self.websocket_url!r}')
+                set_ws(await stack.enter_async_context(session.ws_connect(self.websocket_url)))
+            finally:
+                start_result.set_result(self._ws is not None)
+            stack.callback(set_ws, None)
```
Comment (TheJJ) [inline stustapay/tse/diebold_nixdorf_usb/handler.py:50]: `stack.callback(lambda: self._ws = None)`?

File: stustapay/tse/diebold_nixdorf_usb/handler.py [hunk 2]
```diff
@@ -15,33 +16,58 @@
...
 
-    async def request(self, command: str, *, ws, timeout: float = 5, **kwargs) -> dict:
+            receive_task = asyncio.create_task(self.recieve_loop())
+            await self.request("SetDefaultClientId", ClientID="DummyDefaultClientId")
+            while not self._stop:
+                print('sending pingpong')
+                result = await self.request("PingPong")
```
Comment (TheJJ) [inline stustapay/tse/diebold_nixdorf_usb/handler.py:56]: this pingpong checks for service liveliness, right? what happens when it's no longer alive?

**Review by TheJJ — APPROVED** [review]
pls fix linting

File: stustapay/tse/diebold_nixdorf_usb/handler.py (hunk 1 above)
Comment (mic-e) [reply stustapay/tse/diebold_nixdorf_usb/handler.py]: lambdas cannot have statements

File: stustapay/tse/diebold_nixdorf_usb/handler.py (hunk 2 above)
Comment (mic-e) [reply stustapay/tse/diebold_nixdorf_usb/handler.py]: repsonse timeout -> exception

**Pull request merged**